src/screens/Home/Home.tsx

```diff
@@ -39,39 +59,113 @@ function Home(props: HomeProps) {
+  const amountOwed = isOnline ? 6000 : 0;
+
+  const getAmountOwedCard = () => {
+    const amountOwedCardInfo: CardPropsInfo[] = [{
+      number: amountOwed.toString(),
+      label: 'Owed to MP',
+      unit: 'Ks'
+    }]
+
+    const addButton = <IconButton
+      component={Link}
+      to='/home'
+      size="small"
+      className={classes.buttonPrimary}
+    >
+      <AddIcon />
+    </IconButton>
+
+    return (
+      // Same margin bottom that Typography applies as gutterBottom
+      <div style={{ marginBottom: '.35em' }}>
+        <OutlinedCardList
+          highlighted={amountOwed !== 0}
+          info={amountOwedCardInfo}
+          rightIcon={amountOwed === 0 ? <CheckCircleOutlineIcon /> : addButton}
+        />
+      </div>)
+  }
+
   return (
     <BaseScreen rightIcon="user">
-      <div className={classes.header}>
-        <SiteMenu currentSite={currentSite} sites={allSites} />
-        <div className={classes.network}>
-          {isOnline ? <CloudQueueIcon color="primary" /> : <CloudOffIcon color="secondary" />}
-          <Typography className={classes.network} variant="body1">
-            Last Connected to Network <br />
-            {lastUpdated}
-          </Typography>
+      <BaseScrollView>
+        <div className={classes.header}>
+          <div style={{ flex: 2 }}>
+            <SiteMenu currentSite={currentSite} sites={allSites} />
+            <Typography color='secondary'>
+              Current Period: {formatDateStringToLocal(getCurrentPeriod(), true)}
+            </Typography>
+            <Typography color='secondary'>
+              Deadline: {formatDateStringToLocal(getNextPeriod(), true)}
+            </Typography>
+          </div>
+          <div className={classes.network}>
+            {isOnline ? <CloudOutlinedIcon /> : <CloudOffIcon color="disabled" />}
+            <Typography variant="caption" color='secondary'>
+              Last Connected to Network
+            </Typography>
+            <Typography variant="caption" color='secondary'>
+              {lastUpdated}
+            </Typography>
+          </div>
         </div>
-      </div>
 
-      <Link to={'/customers'}>
-        <HomeMenuItem
-          label="Customer Alerts"
-          amount={numCustomersToMeter + numCustomersToCollect}
-          sublabels={[
-            { amount: numCustomersToMeter, label: 'Number of customers to meter' },
-            { amount: numCustomersToCollect, label: 'Number of customers to collect' },
-          ]}
-        />
-      </Link>
-      <Link to={'/financial-summary'}>
-        <HomeMenuItem label="Unpaid Reports" amount={0} />
-      </Link>
-      <HomeMenuItem label="Unresolved Incidents" amount={0} />
-      <Link to={'/financial-summary'}>
-        <HomeMenuItem label="Financial Summary" noBadge={true} />
-      </Link>
+        {user?.organization === 'Meepanyar' && getAmountOwedCard()}
+
+        <Typography variant='h2' gutterBottom>
```

Comment:
I appreciate that you're using the built-in Material-UI props but the default spacing here is pretty small and doesn't really match the designs (`gutterBottom` makes it only ~6px). Even though it means more containers, I'd wrap the contents of this list (not incl. this heading) and add a `marginTop = theme.spacing(2)` which will make it look more comfortable!

Reply:
Changed so that the margins are consistent, thanks for pointing this out!